src/javascript/Edit/copyLanguage/copyLanguage.utils.spec.js

```diff
@@ -16,4 +16,45 @@ describe('Copy language Utils', () => {
         expect(getFullLanguageName(languages, 'fr').label).toEqual(languages[0].label);
         expect(getFullLanguageName(languages, 'en').label).toEqual(languages[1].label);
     });
+
+    it('test getI18nFieldAndValues', () => {
```

Comment:
Can you improve a bit the test to check its results, not only the number of expected results
And also test with no results.